components/script/dom/messageevent.rs

```diff
@@ -136,24 +136,18 @@ impl MessageEvent {
         messageevent.upcast::<Event>().fire(target);
     }
 
-    pub fn dispatch_error(
-        target: &EventTarget,
-        scope: &GlobalScope,
-        message: HandleValue,
-        origin: Option<&str>,
-        source: Option<&WindowProxy>,
-        ports: Vec<DomRoot<MessagePort>>,
-    ) {
+    pub fn dispatch_error(target: &EventTarget, scope: &GlobalScope, source: Option<&WindowProxy>) {
```

Comment:
The default source value is null, so we don't need to take an argument for it here.